public/app/core/components/Login/LoginLayout.tsx

```diff
@@ -92,90 +92,90 @@ export const getLoginStyles = (theme: GrafanaTheme2) => {
       alignItems: 'center',
       justifyContent: 'center',
     }),
-    loginAnim: css`
-      &:before {
-        opacity: 1;
-      }
+    loginAnim: css({
+      ['&:before']: {
+        opacity: 1,
+      },
 
-      .login-content-box {
-        opacity: 1;
-      }
-    `,
-    submitButton: css`
-      justify-content: center;
-      width: 100%;
-    `,
-    loginLogo: css`
-      width: 100%;
-      max-width: 60px;
-      margin-bottom: 15px;
+      ['.login-content-box']: {
+        opacity: 1,
+      },
+    }),
+    submitButton: css({
+      justifyContent: 'center',
+      width: '100%',
+    }),
+    loginLogo: css({
+      width: '100%',
+      maxWidth: 60,
+      marginBottom: 15,
```

Comment:
Is `marginBottom` equal to `15px`? Shouldn't we use `theme.spacing(2)`, although it'll be `16px` instead of `15px` 🤔